lib/Sema/SemaExpr.cpp

```diff
@@ -12359,6 +12359,50 @@ ExprResult Sema::ActOnRangeBoundsExpr(SourceLocation BoundsKWLoc,
                                        RParenLoc);
 }
 
+ExprResult Sema::CreateRangeBoundsExpr(SourceLocation BoundsKWLoc,
```

Comment:
The reason I wrote is that it is used in RebuildRangeBoundsExpr() in TreeTransform.h. and ActOnRangeBoundsExpr is called in ParseBoundsExpression. When I looked into the source code, I thought the CreateBoudsInteropType is kind of CerateRangeBoundsExpr. In the CreateRangeBoundsExpr, ActOnRangeBoundsExpr is called and the RelativeBoundsClause is setted.